tests/test_dac.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
```

Comment:
While I "covered" all the functions in the `dac.py` module these tests are just a first iteration and pretty lacking to be honest. Just want to have something and avoid huge regressions. As the package evolves we should increase the quality and coverage of the tests.